api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -17,6 +17,8 @@
 import org.openmrs.api.context.Context;
 import org.openmrs.test.BaseContextSensitiveTest;
 
+import junit.framework.Assert;
```

Comment:
@jwnasambu this is the old version of the testing library Junit we use. Please import from package `org.junit`